flink-streaming-java/src/test/java/org/apache/flink/streaming/runtime/tasks/TwoInputStreamTaskTest.java

```diff
@@ -623,36 +597,19 @@ public void testHandlingEndOfInput() throws Exception {
 		testHarness.invoke();
 		testHarness.waitForTaskRunning();
 
-		TestBoundedTwoInputOperator headOperator = (TestBoundedTwoInputOperator) testHarness.getTask().headOperator;
-
```

Comment:
Is this test simplification related to the rest of the commit? Was this test failing in the original form?

Reply:
> Was this test failing in the original form?